cmd/object-handlers-common_test.go

```diff
@@ -51,3 +57,64 @@ func TestCanonicalizeETag(t *testing.T) {
 		}
 	}
 }
+
+// Tests - CheckPreconditions()
+func TestCheckPreconditions(t *testing.T) {
+	objModTime := time.Date(2024, 8, 26, 02, 01, 01, 0, time.UTC)
+	t.Run("If-None-Match", func(t *testing.T) {
```

Comment:
The usual pattern with t.Run is to call it within the range over testCases on line 78, rather than wrapping the entire block. That way each testCase is it's own sub-run and reports individual results.